@navikt/core/react/src/accordion/AccordionItem.tsx

```diff
@@ -65,6 +65,7 @@ const AccordionItem = forwardRef<HTMLDivElement, AccordionItemProps>(
           "navds-accordion__item--neutral": context?.variant === "neutral",
           "navds-accordion__item--no-animation": !shouldAnimate.current,
         })}
+        data-expanded={_open}
```

Comment:
Why not use `.navds-accordion__item--open`?

Reply:
I saw this pattern used a lot in Radix, and quite like what it achieves:
- User can always see the current state in the dom
- Using classes always ends with some instances being "positive" and some "negative", so --closed or --open. Using `data-<state>` removes that issue  

This is mostly relevant for the cases where the component has styles applied for both states

Reply:
I super support this moving of state classes -> data-<state> attributes 🎉 . I don't like the "negative/positive" mix problem in css classes that inevitably happens 😩 .